examples/README.md

```diff
@@ -92,318 +92,318 @@ git checkout v0.4.0
 <!-- markdownlint-disable-next-line MD026 -->
 ## Hello, World!
 
-Example | Description
---- | ---
-[`hello_world.rs`](./hello_world.rs) | Runs a minimal example that outputs "hello world"
+|Example | Description|
+|--- | ---|
+|[`hello_world.rs`](./hello_world.rs) | Runs a minimal example that outputs "hello world"|
 
 # Cross-Platform Examples
 
 ## 2D Rendering
 
-Example | Description
```

Comment:
completely unnecessary changes here revert the entire file

Reply:
Our markdown style is to have the tables with that style, so this is just fixing that. It is even inconsistent within the file itself.

Reply:
I can make it its own PR if you'd rather.

Reply:
does it render the same way? i think it looks noisy. can we standardize the other way?

Reply:
It does render the same way. I prefer with the bars, and I think that's what is more often used in general. I can change it if a decision-maker wants me to. We only have 2 .md files with tables.